tests/stdlib/experimental/tcmdline.nim

```diff
@@ -0,0 +1,2191 @@
+discard """
+description: "Tests for the cmdline module"
+targets: "c js vm"
+knownIssue.vm: "tables crashes with 'trying to access location outside of VM's memory'"
```

Comment:
```suggestion
  description: "Tests for the cmdline module"
  targets: "c js vm"
  knownIssue.vm: '''
    `.tailcall` in combination with `sink` is not supported due to `vmgen` bug,
    causing an error at run-time 
  '''
```
This make the known-issue description more helpful to someone wanting to look into the issue.